src/DearImGui.hs

```diff
@@ -390,6 +393,24 @@ colorPicker3 desc ref = liftIO do
 
     return changed
 
+-- | SliderFloat
+--
+-- Wraps @ImGui::SliderFloat()@
```

Comment:
For the documentation, I've been copied (literally) what's in [`imgui.h`](https://raw.githubusercontent.com/ocornut/imgui/master/imgui.h). For `SliderFloat` there isn't any documentation, so let's go with just:

```suggestion
-- | Wraps @ImGui::SliderFloat()@
```